### DISCUSSION TITLE
请问为什么在本地能够正常查看，但是hexo d之后，就显示不出来呀

### DISCUSSION THREAD

**Discussion by EdgarSllow** [YunYouJun/hexo-theme-starry#339] [General 大局为重]
如图
![image](https://user-images.githubusercontent.com/86147832/176355432-23002d95-fd56-441e-b2d3-f3fa27d16c96.png)

**Comment by EdgarSllow** [comment]
localhost就ok
![image](https://user-images.githubusercontent.com/86147832/176355790-4796288f-eb29-4b80-a948-b78bfe1cc9b6.png)

**Comment by YunYouJun** [comment]
请提供可复现仓库或预览地址。

**Comment by Tru-tru** [comment]
楼主你好，
我遇到了和你一样的问题。
刚刚看你博客问题已经解决了，请问你是怎么解决的？

**Comment by YunYouJun** [comment]
多半是路径/CDN 问题。
